src/appengine/handlers/cron/grouper.py

```diff
@@ -32,9 +33,12 @@
 
 GROUP_MAX_TESTCASE_LIMIT = 25
 
-VARIANT_CRASHES_IGNORE = ['Out-of-memory', 'Timeout']
+VARIANT_CRASHES_IGNORE = re.compile(
+    r'^Out-of-memory|^Timeout|^Missing-library|^Data race')
```

Comment:
nit: do

```
r'^(Out-of-memory|...|...)'
```

to avoid many repetitions of `^`.

Reply:
Oh! nice!